.github/workflows/build.yml

```
@@ -22,8 +22,8 @@ jobs:
       - name: Install cross-compilers
         run: sudo apt-get update && sudo apt-get install -y gcc-mingw-w64
 
-      - name: Cache Go modules
-        uses: actions/cache@v4
+      - name: Setup Parser
```

actionlint (1.7.7)

step must run script with "run" section or run action with "uses" section

(syntax-check)